webpack/JobWizard/JobWizard.js

```diff
@@ -116,23 +126,32 @@ export const JobWizard = () => {
           jobTemplateID={jobTemplateID}
         />
       ),
-      canJumpTo: isTemplate,
+      canJumpTo: isTemplate && valid.hostsAndInputs,
+      enableNext: isTemplate && valid.hostsAndInputs && valid.advanced,
```

Comment:
If the user got here, it probably means that `valid.hostsAndInputs` is true.

Reply:
But a user can go a step back and enter invalid values